$NetBSD: patch-ab,v 1.3 2010/06/03 05:24:27 dholland Exp $

Fix default path lengths too short for it to hold its own include path.

--- src/header.h.dist	2007-02-04 18:15:15.000000000 -0500
+++ src/header.h	2007-02-04 18:17:15.000000000 -0500
@@ -342,6 +342,7 @@
 #define FN_SEP '/'
 /* 5 */
 #define Temporary_Directory "/tmp"
+#define PATHLEN 1024
 #define INCLUDE_TASK_ID
 #define _POSIX_C_SOURCE 199506L
 #define _XOPEN_SOURCE 500
@@ -403,6 +404,7 @@
 #define FN_SEP '/'
 /* 5 */
 #define Temporary_Directory "/tmp"
+#define PATHLEN 1024
 #define INCLUDE_TASK_ID
 #ifdef MAIN_INFORM_FILE
 static int32 unique_task_id(void)
@@ -424,6 +426,7 @@
 #define FN_SEP '/'
 /* 5 */
 #define Temporary_Directory "/tmp"
+#define PATHLEN 1024
 #define INCLUDE_TASK_ID
 #ifdef MAIN_INFORM_FILE
 static int32 unique_task_id(void)
